src/main/java/assets/game.js

```diff
@@ -24,8 +24,12 @@ function markHits(board, elementId, surrenderText) {
             className = "miss";
         else if (attack.result === "HIT")
             className = "hit";
-        else if (attack.result === "SUNK")
-            className = "hit"
+        else if (attack.result === "SUNK") {
+            className = "sink";
```

Comment:
Might need to refactor the code later to change the class names in the project to sunk...less confusing that way.